pkg/cloud/vsphere/services/cloudprovider/csi.go

```diff
@@ -343,7 +353,7 @@ func LivenessProbeForNodeContainer(image string) corev1.Container {
 	}
 }
 
-func CSIControllerStatefulSet(storageConfig cloudprovider.StorageConfig) *appsv1.StatefulSet {
+func CSIControllerStatefulSet(storageConfig *cloudprovider.StorageConfig) *appsv1.StatefulSet {
```

Comment:
Why pass by reference? Unless changes are made, passing by copy is actually a lot easier on Go.